renderlib/VolumeDimensions.h

```diff
@@ -14,6 +14,10 @@ struct VolumeDimensions
   float physicalSizeY = 1.0f;
   float physicalSizeZ = 1.0f;
   uint32_t bitsPerPixel = 16;
+  // SAMPLEFORMAT_UINT = 1;
```

Comment:
What's this about?

Reply:
sampleFormat basically keeps track of whether the original source data was float or uint

Reply:
in this case I am using the libtiff-specific values, so if I make changes for reading CZI files I'll have to reconcile things at that time.